app/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Providers\RouteServiceProvider;
+use Illuminate\Foundation\Auth\AuthenticatesUsers;
+use Illuminate\Http\Request;
+
+class LoginController extends Controller
+{
+    /*
+    |--------------------------------------------------------------------------
+    | Login Controller
+    |--------------------------------------------------------------------------
+    |
+    | This controller handles authenticating users for the application and
+    | redirecting them to your home screen. The controller uses a trait
+    | to conveniently provide its functionality to your applications.
+    |
+    */
+
+    use AuthenticatesUsers;
+
+    /**
+     * Where to redirect users after login.
+     *
+     * @var string
+     */
+    protected $redirectTo = RouteServiceProvider::HOME;
+    protected function redirectTo(){
+      if (Auth()->user()->role == 1){
+        return route('admin.dashboard');
```

Comment:
buradaki mantık çok yanlış. 2 tane dashboard'ın olmayacak. Admin için ayrı user için ayrı sayfalar yapmicaksın. Bir tane dashboard olarak içerisinde admin ise şunlarıda görsün şeklinde bir yapı oluşturman lazım. Yani if(Auth()->user()->role == 1) ise admin oluyor diyelim belli şeyleri görsün gibi.